helm-chart/renku/requirements.yaml

```diff
@@ -39,3 +39,7 @@ dependencies:
   version: 10.7.11
   repository: "https://raw.githubusercontent.com/bitnami/charts/eb5f9a9513d987b519f0ecd732e7031241c50328/bitnami"
   condition: redis.install
+- name: rabbitmq
+  version: 11.2.2
+  repository: https://charts.bitnami.com/bitnami
```

Comment:
We should not use this repo URL, otherwise, it'll be unavailable in 6 months. See Redis above for the 'correct' way of specifying bitnami repositories. Better yet would be to target the chart tarball file.